synapse/models/infotech.py

```diff
@@ -1058,9 +1058,16 @@ def getModelDefs(self):
                     ('software:name', ('str', {'lower': True, 'strip': True}), {
                         'doc': 'The name of the software at a particular version.',
                     }),
+                    ('name', ('str', {'lower': True, 'strip': True}), {
```

Comment:
Name should be an `it:dev:str` similar to the array for now

Reply:
updated